common/buildings/wc_common_buildings.txt

```diff
@@ -0,0 +1,682 @@
+﻿### aviary complex
```

Comment:
Errors:
```
	Line 978: " in file: "common/buildings/wc_common_buildings.txt" near line: 417
	Line 980: " in file: "common/buildings/wc_common_buildings.txt" near line: 454
	Line 982: " in file: "common/buildings/wc_common_buildings.txt" near line: 496
	Line 984: " in file: "common/buildings/wc_common_buildings.txt" near line: 538
	Line 986: " in file: "common/buildings/wc_common_buildings.txt" near line: 580
	Line 988: " in file: "common/buildings/wc_common_buildings.txt" near line: 625
	Line 998: " in file: "common/buildings/wc_duchy_capital_buildings.txt" near line: 64
	Line 1000: " in file: "common/buildings/wc_duchy_capital_buildings.txt" near line: 115
	Line 1002: " in file: "common/buildings/wc_duchy_capital_buildings.txt" near line: 168
	Line 1004: " in file: "common/buildings/wc_duchy_capital_buildings.txt" near line: 218
	Line 1006: " in file: "common/buildings/wc_duchy_capital_buildings.txt" near line: 267
	Line 1008: " in file: "common/buildings/wc_duchy_capital_buildings.txt" near line: 315
```

Reply:
Cant check if this is fixed, but I found a parenthesis error, so it should have fixed